tests/cdn/loadCKEditorCloud.test.ts

```diff
@@ -78,6 +81,72 @@ describe( 'loadCKEditorCloud', () => {
 		expect( loadedPlugins?.Plugin ).toBeDefined();
 	} );
 
+	describe( 'CSP', () => {
+		it( 'should set crossorigin=anonymous attribute on injected elements if `htmlAttributes` is not specified', async () => {
+			await loadCKEditorCloud( {
+				version: '43.0.0'
+			} );
+
+			expect( queryNonAnonymousLinks() ).toHaveLength( 0 );
+			expect( queryNonAnonymousScripts() ).toHaveLength( 0 );
+		} );
+
+		it( 'should set crossorigin=anonymous attribute on injected elements if `htmlAttributes` is specified but is blank', async () => {
+			await loadCKEditorCloud( {
+				version: '43.0.0',
+				injectedHtmlElementsAttributes: {}
+			} );
+
+			expect( queryNonAnonymousLinks() ).toHaveLength( 0 );
+			expect( queryNonAnonymousScripts() ).toHaveLength( 0 );
+		} );
+
+		it( 'should be possible to override the `crossorigin` attribute', async () => {
+			const promise = loadCKEditorCloud( {
+				version: '43.0.0',
+				injectedHtmlElementsAttributes: {
+					crossorigin: 'use-credentials'
+				}
+			} );
+
+			// It's fine because `use-credentials` throws an error in the vitest browser.
+			expect( promise ).rejects.toThrowError();
+		} );
```

Comment:
Missing check if the `crossorigin: 'use-credentials'` attribute is set in scripts and links. `document.querySelectorAll()` should work even if error is thrown.

Reply:
From what I can see, it crashes before appending it to document @psmyrek. I'll check if we can test it better

Reply:
@psmyrek I added spy that checks `setAttribute`, should be enough.